source/renderer/app/components/profile/bug-report/BugReportDialog.js

```diff
@@ -145,18 +147,26 @@ export default class BugReportDialog extends Component<Props, State> {
 
   state = {
     showLogs: true,
+    isReadyToSubmit: false,
+    wasSubmitted: false,
+    compressedLog: null
   };
 
   componentWillMount() {
-    this.props.onGetLogs();
-    this.props.onDeleteCompressedLogs();
+    this.props.onGetFreshLogs();
```

Comment:
@daniloprates we need to make sure this call only fetches the log files list and doesn't compress them!

Reply:
Even when the user submits it?